rllib/core/rl_module/tests/test_marl_module.py

```diff
@@ -14,13 +15,14 @@
 
 class TestMARLModule(unittest.TestCase):
     def test_from_config(self):
-
+        """Tests whether a MultiAgentRLModule can be constructed from a config."""
```

Comment:
Can we create a a separate test unit for this? 

I want to leave this method as a simple unittest without dependency on AlgorithmConfig.

Reply:
Removed, this is now covered in TestAlgorithmConfig in a separate test.